test/IniModule.test.cpp

```diff
@@ -0,0 +1,53 @@
+#include "FileSystemHelper.h"
+#include "FsInclude.h"
+#include "IniFile.h"
+#include "ModuleLoader.h"
+#include <fstream>
+#include <string>
+#include <string_view>
+#include <gtest/gtest.h>
+
+
+// sectionPairs is a non-line terminated list of keys and values. EG: TestModule = No
+void WriteExternalModuleIniFile(std::string_view iniFilename, const std::vector<std::string_view> sectionPairs)
+{
+	std::ofstream iniFileStream(iniFilename);
+
+	iniFileStream << "[ExternalModules]\n";
+
+	for (const auto& sectionPair : sectionPairs)
+	{
+		iniFileStream << sectionPair << "\n";
+		//iniFileStream << "Test = yes\n";
```

Comment:
Maybe get rid of this commented out line.

Reply:
Thanks for catching my sloppiness here.